modules/cli-installing-cli-rpm.adoc

```diff
@@ -43,20 +43,16 @@ For {op-system-base-full}, you can install the OpenShift CLI (`oc`) as an RPM if
 ----
 
 . Enable the repositories required by {product-title} {product-version}.
-
-** For Red Hat Enterprise Linux 9:
-+
-[source,terminal]
-----
-# subscription-manager repos --enable="rhocp-4.11-for-rhel-9-x86_64-rpms"
-----
-
-** For Red Hat Enterprise Linux 8:
 +
 [source,terminal]
 ----
 # subscription-manager repos --enable="rhocp-4.11-for-rhel-8-x86_64-rpms"
 ----
++
+[NOTE]
+====
+It is not supported to install the OpenShift CLI (`oc`) as an RPM for {op-system-base-full} 9. You must install the OpenShift CLI for {op-system-base} 9 by downloading the binary.
```

Comment:
I guess same comment as in https://github.com/openshift/openshift-docs/pull/47922/files#r923681274 applies.

Reply:
Unfortunately we can't link to other sections of the docs in our modules :slightly_frowning_face: But in this case, the docs for installing using other methods are right on this same page, so it shouldn't be an issue